test/integration/content-sidebar/ContentSidebar.e2e.test.js

```diff
@@ -74,7 +74,7 @@ describe('ContentSidebar', () => {
     describe('version history', () => {
         beforeEach(() => {
             helpers.load({
-                features: { versions: true },
+                hasVersions: true,
```

Comment:
Still needed? This is already set as the default value on the ContentSidebar example.